src/main/java/com/mhrglobal/print/PrintService.java

```diff
@@ -1,14 +1,5 @@
 package com.mhrglobal.print;
 
-//Do not change
```

Comment:
I try to not have comments on the code to make it cleaner and easy to read.

I like Extreme programming and clean code. If I need a comment it means to me that the code it is not expressing on its own. Code is dynamic and constantly changing. If change the code I need to change my comments.